editor/editor_node.cpp

```diff
@@ -6704,6 +6704,9 @@ EditorNode::EditorNode() {
 	screenshot_timer->connect("timeout", callable_mp(this, &EditorNode::_request_screenshot));
 	add_child(screenshot_timer);
 	screenshot_timer->set_owner(get_owner());
+
+	String exec = OS::get_singleton()->get_executable_path();
```

Comment:
Might be worth adding a comment, e.g. `// Save editor executable path for thirdparty tools.`.
This could also be `const String &`.